enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -284,6 +293,28 @@ func (l *FileCacheLoader) CacheSnapshot(ctx context.Context, key *fcpb.SnapshotK
 	return &Snapshot{key: key, manifest: manifest}, nil
 }
 
+func (l *FileCacheLoader) checkAllArtifactsExist(ctx context.Context, manifest *fcpb.SnapshotManifest) error {
```

Comment:
can't wait till this can just be a call to GetActionResult

Reply:
agreed - maybe another good reason to migrate filecache -> disk_cache and remove filecache? since we'd need a local AC for that to work